code/game/gamemodes/changeling/changeling.dm

```diff
@@ -75,7 +75,7 @@ GLOBAL_VAR(changeling_team_objective_type) //If this is not null, we hand our th
 	for(var/datum/mind/changeling in changelings)
 		log_game("[key_name(changeling)] has been selected as a changeling")
 		var/datum/antagonist/changeling/new_antag = new()
-		new_antag.team_mode = TRUE
+		new_antag.team_mode = FALSE //Should stop team lings from spawning

```

Comment:
If you're removing team lings, you should remove all of the relevant code from this proc.

Reply:
I'd not call this a full removal, rather simply disabling teamlings. Some people, though they are a minority, do enjoy them, and a time may come in which team lings are actually fair and balanced.